### PR TITLE
Cape Town 2016

### PR DESCRIPTION
**Pull request by cobusbernard**
Bank account open, venue secured, basic details in. Registration is still in progress, arrange with local ticket provider as Eventbrite do not do Africa :|

### CODE REVIEW

File: content/events/2016-capetown/sponsor.md [hunk 1]
```diff
@@ -1,49 +1,195 @@
...
 title = "sponsor"
 type = "event"
 
-
 +++
 
-We greatly value sponsors for this open event.  If you are interested in sponsoring, please drop us an email at [{{< email_organizers >}}].
-
-<hr>
+Thank you for being here, and for considering sponsorship of Cape Town's first DevOpsDays! Sponsors are what makes community events like this possible. If you are interested in sponsoring, please drop us an email at {{< email_organizers >}}. Gold, Silver, and Bronze packages are detailed in our <a href="https://dl.dropboxusercontent.com/u/443584/DevOpsDays/devopsdays-nashville-2016-prospectus.pdf">prospectus</a>.
```
Comment (kmugrage) [inline content/events/2016-capetown/sponsor.md]: You have a link to the Nashville prospectus here

File: content/events/2016-capetown/sponsor.md [hunk 2]
```diff
@@ -1,49 +1,195 @@
...
+        <li>Shoutout via Twitter (<a href="https://twitter.com/devopsdayscpt">@DevOpsDaysCPT</a>)</li>        
+      </ul>
+  </td>
+  <td>
+    <i>Contact us! {{< email_organizers >}}</i>
+  </td>
+  </tr>
+</table>
+
+### Conference wrap-up Social – Availability 1 (R30,000 ~ R35,000)
```
Comment (adrianmoisey) [inline content/events/2016-capetown/sponsor.md]: "Conference wrap-up" sounds like it's for the end of the last day at the conf. What about changing the wording to something like "Conference day 1 wrap-up Social" ?

Same for the other day.

Comment (cobusbernard) [inline content/events/2016-capetown/sponsor.md]: Thanks, fixing this.

**Comment by cobusbernard** [comment]
Last changes are in until we have the ticket sales set up, please merge when ready.

**Pull request merged**